RELEASE/scripts/autoscend.ash

```diff
@@ -1722,6 +1724,7 @@ boolean doTasks()
 	auto_CMCconsult();
 	auto_checkTrainSet();
 	auto_autumnatonQuest();
+	auto_SITCourse();
```

Comment:
With my comments in `auto_SITCourse()` believe this can get moved by `pickRocks()` as it should be a 1/day thing

Reply:
Makes sense. I think I had it there because of the flapper fly buff but since I removed that into auto_post_adv, I have moved it